src/Helper/ExceptionConverter.php

```diff
@@ -44,11 +46,22 @@ public static function convertExceptionToResult(
         $result->setMessage($transformedException->getError())
             ->setErrorType($errorType)
             ->setExceptionId($transformedException->getExceptionId());
+
+        $variables = new VariableCollection();
+
         if ($outputs) {
             $result
                 ->setConfigVersion((string) $outputs[0]->getConfigVersion())
                 ->setImages(array_map(fn(Output $output) => $output->getImages(), $outputs));
+
+            foreach ($outputs as $output) {
+                foreach ($output->getInputVariableValues() as $variableName => $variableValue) {
+                    $variables->addVariable(new Variable((string) $variableName, $variableValue));
+                }
+            }
         }
+
+        $result->setVariables($variables);
```

Comment:
Nemuze to byt vsechno v tom ifu? 
I `$variables = new VariableCollection()` i `$result->setVariables($variables);`

Nebo to pak v tom resultu neni inicializovane?